### ISSUE TITLE
Better handling of images where the contributor code needs additional, individual information

### ISSUE THREAD

**Issue by alexwlchan** [wellcometrust/platform#1152]
We have a handful of images on the transformer DLQ that are either in the wrong place, or need a transformer fix. We'll need to talk to Christy/Jenny to find out where they go, in some cases.

*   [ ] **image_use_restrictions = "Do not use"**
  B0008795, B0008796, B0008797, B0008798, B0008799, N0037137, N0037139, N0037141

* [ ] **image_use_restrictions = "Image withdrawn, see notes"**
  B0008069, B0008070, B0008071, B0008072, B0008073, N0036283, N0036284, N0036300, N0036301, N0036302, N0036303

* [ ] **image_use_restrictions = null**
  B0007773, N0033141, N0036049, W0026564, W0026884

* [ ] **image_use_restrictions = "See Copyright Information"**
  B0010525

* [ ] **image_innopac_id = "L 35411\n15551040**
  L00035411

**Comment by alexwlchan** [comment]
Currently DLQ state:

B0006507
B0006714
B0007773
B0007831
B0007832
B0007833
B0007834
B0007835
B0007836
B0008069
B0008070
B0008071
B0008072
B0008073
B0008795
B0008796
B0008797
B0008799
B0010525
B0011276
B0011277
B0011278
B0011302
B0011303
L0035411
N0033141
N0036049
N0036283
N0036284
N0036300
N0036301
N0036302
N0036303
N0037137
N0037139
N0037141
N0038265
N0038266
W0026564
W0026884

**Comment by alexwlchan** [comment]
So we have a bunch of individuals for whom the contributor code maps to something like “individual scientists” or “TBC”, which I scrubbed from the contributor map, but then the transformer fails with a “could not find key” error.

We also have credit lines with text like:

> [scientist's name], University of Lincoln

where currently the literal `[scientist's name]` is being dropped before we expose the credit line. It would be nice to plumb that through from the image as well.

**Comment by alexwlchan** [comment]
The current images are all failing because the transformer can't find a contributor credit line for them:

```
{'CHC': ['B0007831',
  'B0007832',
  'B0007833',
  'B0007834',
  'B0007835',
  'B0007836'],
 'CSC': ['B0006507', 'B0006714'],
 'FDN': ['B0011306', 'B0011307', 'B0011308'],
 'JAC': ['B0011004'],
 'MKK': ['B0011276', 'B0011277', 'B0011278'],
 'RHL': ['B0011302', 'B0011303'],
 'RSM': ['B0011149'],
 'ZGS': ['B0011321', 'B0011322']}
```

We'll probably need to do these on a case-by-case basis when we go through the snagging list.

**Comment by alexwlchan** [comment]
Closed by #2213.
